Extension/src/LanguageServer/settings.ts

```diff
@@ -30,7 +31,14 @@ export class CppSettings extends Settings {
         super("C_Cpp", resource);
     }
 
-    public get clangFormatPath(): string { return super.Section.get<string>("clang_format_path"); }
+    public get clangFormatPath(): string {
```

Comment:
This will fix the initial configuration sent to the extension, but will it work if someone changes the setting and then clears it again? I don't believe the settings notifications that come from VS Code and go to the language server call into this function.

Reply:
In other words, you might need to communicate the clangFormatPath that comes from $PATH down to the language server.

Reply:
Code just needs to be added near where it sends the UpdateClangFormatPathNotification in the Typescript.

Reply:
Oh, no, it looks fine -- the update code calls into settings.clangFormatPath.

Reply:
I forgot we already had a notification for this.  Thanks for checking!